Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,5 +1,47 @@
-function setAlarm() {}
+function setAlarm() {
+  const alarmTimeInput = document.getElementById("alarmSet");
+  const alarmTimeInSeconds = alarmTimeInput.value;
+  let totalSeconds = parseInt(alarmTimeInSeconds);
```

Comment:
These are much clearer names, but now I'm not sure the difference between `alarmTimeInSeconds` and `totalSeconds` - can you differentiate these better too? :)